src/components/movie/AddMovies.jsx

```diff
@@ -1,5 +1,5 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
[Optional]
- [ ] You should try to fix the issue instead of surprising the linter errors. Working on fixing linter errors will give you deeper understanding on coding conventions and make your code more professional